connect/transforms/src/main/java/org/apache/kafka/connect/transforms/TimestampConverter.java

```diff
@@ -271,16 +277,18 @@ public Date toType(Config config, Date orig) {
     // This is a bit unusual, but allows the transformation config to be passed to static anonymous classes to customize
     // their behavior
     private static class Config {
-        Config(String field, String type, SimpleDateFormat format, String unixPrecision) {
+        Config(String field, String type, SimpleDateFormat format, String unixPrecision, String time_zone) {
```

Comment:
```suggestion
        Config(String field, String type, SimpleDateFormat format, String unixPrecision, String timeZone) {
```

Please keep casing consistent :)

Reply:
Adjusted the code based on feedback，please review again，thank you very much ！